website/cue/reference/components.cue

```diff
@@ -618,6 +618,15 @@ components: {
 							examples: ["/path/to/certificate_authority.crt"]
 						}
 					}
+          client_metadata_key: {
+						common:      false
+						description: "The key name added to each event with the client certificate's metadata."
```

Comment:
I neglected this before, but should we add some notes about what will be added to the event when this key is set? Components have output descriptions too, like https://vector.dev/docs/reference/configuration/sources/socket/#output-logs , which seemingly it'd be nice to integrate this with.

Reply:
Mhmm, your choice on if this is blocking - I'd like to move onto the ack work I have, but don't want the docs to rot